identity/app/views/profileForms.scala.html

```diff
@@ -96,7 +97,7 @@ <h1 class="identity-title" data-test-id="edit-profile-header">Edit your profile<
                     <div class="tabs__content">
                         @content(1, "/public/edit")(profile.publicProfileForm(idUrlBuilder, idRequest, user, forms.publicForm))
 
-                        @content(2, "/account/edit")(profile.accountDetailsForm(idUrlBuilder, idRequest, user, forms.accountForm))
+                        @content(2, "/account/edit")(profile.accountDetailsForm(idUrlBuilder, idRequest, user, forms.accountForm, changedEmail = changedEmail))
```

Comment:
Just `changedEmail` instead of `changedEmail = changedEmail` should work.

Reply:
Cheers